### ISSUE TITLE
Error when a underscore is present in the parent directory of the project

### ISSUE THREAD

**Issue by GlennFranquet** [irok/sprite-magic-importer#5]
All my coding projects are in a directory _projectSites.
So I have for example c:/_projectSites/client1/project1 => here I run the npm command to compile the sass files.

When compiling I get the following error: Path must be a string. Received undefined
Reason for that is that the function isPartialFile in lib >> index.js returns for all the files true.
So the code don't find any file to handle.

The code is working when I remove the underscore in the directory name.

**Comment by RyanMulready** [comment]
There also seems to be a problem with similar image names that have an underscore.  For instance:

image1_active.png
image2_inactive.png

```
.icon1 {
    @include myicons-icons-sprite(image1_active);
    &.is-inactive { @include myicons-icons-sprite(image2_inactive); }
}
```

Will return error

`Error: argument `$map` of `map-get($map, $key)` must be a map`

**Comment by irok** [comment]
@GlennFranquet @RyanMulready Thanks for reporting the issue.
I will fix those problems.

**Comment by irok** [comment]
@GlennFranquet @RyanMulready I have released 1.6.0 which resolved those problems now.
